keel-core/src/test/kotlin/com/netflix/spinnaker/keel/veto/UnhappyVetoTests.kt

```diff
@@ -80,17 +83,17 @@ class UnhappyVetoTests : JUnit5Minutests {
         }
 
         test("unhappy resources are vetoed") {
-          expectThat(subject.check(r).allowed).isEqualTo(false)
+          expectThat(check().allowed).isEqualTo(false)
         }
 
         test("resources are checked once every wait time") {
           unhappyRepository.markUnhappyForWaitingTime(r.id, r.spec.application)
 
-          val response1 = subject.check(r)
+          val response1 = check()
           clock.incrementBy(Duration.ofMinutes(11))
```

Comment:
Just FYI, I added `MutableClock.tickMinutes(minutes)` to make this a bit more convenient.